src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -979,6 +978,77 @@ private static bool HasDuplicateInterfaces(NamedTypeSymbol type, ConsList<TypeSy
                 diagnosticsBuilder.Add(new TypeParameterDiagnosticInfo(typeParameter, new CSDiagnosticInfo(errorCode, containingSymbol.ConstructedFrom(), distinguisher.First, typeParameter, distinguisher.Second)));
                 hasError = true;
             }
+
+            static NullableFlowState getTypeArgumentState(in TypeWithAnnotations typeWithAnnotations)
+            {
+                var type = typeWithAnnotations.Type;
+                if (type is null)
+                {
+                    return NullableFlowState.NotNull;
+                }
+                if (type.IsValueType)
+                {
+                    return type.IsNullableType() ? NullableFlowState.MaybeNull : NullableFlowState.NotNull;
```

Comment:
>IsNullableType [](start = 32, length = 14)

```IsNullableTypeOrTypeParameter```?

Reply:
Good catch, thanks! Fixed in [https://github.com/dotnet/roslyn/pull/46405](https://github.com/dotnet/roslyn/pull/46405).

---
In reply to: [458420274](https://github.com/dotnet/roslyn/pull/45993#discussion_r458420274) [](ancestors = 458420274)